js/script.js

```diff
@@ -1,4 +1,4 @@
-googleMapsSatellite = L.tileLayer(
+const googleMapsSatellite = L.tileLayer(

```

Comment:
Otherwise, ChatGPT 4 explains:

> In JavaScript, when you assign a value to a variable without using var, let, or const, it implicitly creates a global variable. This behavior is called "variable hoisting."
>
> ...
>
> It's important to note that using variables without declaring them with var, let, or const is not recommended. This can lead to unexpected behavior, bugs, and difficulties in maintaining the code. Always declare your variables with the appropriate keyword to avoid these issues.